### ISSUE TITLE
Carryover stops with 'Before first heading' after recent change

### ISSUE THREAD

**Issue by mrvdb** [bastibe/org-journal#204]
After revision https://github.com/bastibe/org-journal/commit/3cfe3445eb6c7110d664bdf16165eb6aa9140108

the daily carryover stops with 'Before first heading'

If I revert the change on line 716 https://github.com/bastibe/org-journal/commit/3cfe3445eb6c7110d664bdf16165eb6aa9140108#diff-91a1ed1bf3b40cbc76c2e32a851e9de9L716

the earlier behaviour is restored (at least in my config)

**Comment by cslux** [comment]
Hey! Do you have a minimal example setup to reproduce it?

**Comment by mrvdb** [comment]
[Christian Schwarzgruber]:
> Hey! Do you have a minimal example setup to reproduce it?

Here's my active config:

```

#+BEGIN_SRC emacs-lisp
(use-package org-journal
 :bind
  (("C-c j"   . 'org-journal-new-entry))
 :init
  (setq
    org-journal-dir
     (expand-file-name (concat org-directory "journal/"))
    org-journal-file-header
     "#+SETUPFILE:~/.emacs.d/org-config.org"

    org-agenda-file-regexp "^[0-9]+\\.org"
    org-journal-file-format  "%Y%m%d.org"
    org-journal-date-format "%A, %e-%m-%Y"
    org-journal-date-prefix "#+TITLE: "
    org-journal-time-format "[%R] "
    org-journal-time-prefix "* "

    org-journal-carryover-items "+carryover|+TODO=\"TODO\""
    org-journal-enable-agenda-integration t
    org-journal-carryover-delete-empty-journal 'ask

    org-journal-enable-encryption nil
    org-crypt-disable-auto-save t)
 :config
   (add-hook 'org-journal-mode-hook
     'turn-on-visual-line-mode)
   (add-hook 'org-journal-mode-hook
     'turn-on-visual-fill-column-mode))
#+END_SRC
```

**Comment by cslux** [comment]
Thanks! Fixed via 08c2015.

**Comment by mrvdb** [comment]
Thanks!
